defi/src/utils/normalizeChain.ts

```diff
@@ -3540,771 +3524,229 @@ export const chainCoingeckoIds = {
+  "bfc": "Bifrost Network",
+  "chainx": "BEVM",
+  "ftn": "Bahamut",
+  "polygon_zkevm": "Polygon zkEVM",
+  "eos_evm": "EOS EVM",
+  "oas": "Oasys",
+  "xpla": "XPLA",
+  "neon_evm": "Neon",
+  "gravitybridge": "GravityBridge",
+  "aura": "Aura Network",
+  "mvc": "MVC",
+  "alv": "ALV",
+  "dsc": "DSC",
+  "pg": "Pego",
+  "shimmer_evm": "ShimmerEVM",
+  "nos": "NOS",
+  "renec": "RENEC",
+  "radixdlt": "Radix",
+  "ethf": "ETHF",
+  "meer": "MEER",
+  "elsm": "Elysium",
+  "eon": "Horizen EON",
+  "chz": "Chiliz",
+  "lightlink_phoenix": "LightLink",
+  "pgn": "PGN",
+  "dash": "Dash",
+  "fsc": "FSC",
+  "new": "Newton",
+  "jbc": "JBC",
+  "cmp": "CMP",
+  "airdao": "AirDAO",
+  "dydx": "dYdX",
+  "btn": "Bitnet",
+  "zeta": "ZetaChain",
+  "area": "Areon Network",
+  "defiverse": "DeFiVerse",
+  "manta_atlantic": "Manta Atlantic",
+  "astrzk": "Astar zkEVM",
+  "inevm": "inEVM",
+  "q": "Q Protocol",
+  "bsquared": "BSquared",
+  "imx": "Immutable zkEVM",
+  "defichain_evm": "DeFiChain EVM",
+  "rss3_vsl": "RSS3",
+  "islm": "HAQQ",
+  "svm": "SatoshiVM",
+  "karak": "K2",
+  "xlayer": "X Layer",
+  "lac": "LaChain Network",
+  "bob": "BOB",
+  "btr": "Bitlayer",
+  "ace": "Endurance",
+  "dfs": "DFS Network",
+  "cyeth": "Cyber",
+  "real": "re.al",
+  "iotaevm": "IOTA EVM",
+  "ailayer": "AILayer",
+  "ox_chain": "OXFUN",
+  "etlk": "Etherlink",
+  "reya": "Reya Network",
+  "cronos_zkevm": "Cronos zkEVM",
+  "band": "BandChain",
+  "immutablex": "Immutable X",
+  "neox": "Neo X Mainnet",
+  "idex": "IDEX",
+  "etn": "Electroneum",
+  "wc": "World Chain",
+  "apechain": "ApeChain",
+  "assetchain": "Asset Chain",
+  "unit0": "UNIT0",
+  "rbn": "Redbelly",
+  "xsat": "exSat",
+  "tara": "Taraxa",
+  "vinu": "VinuChain",
+} as { [key: string]: string }
+
+// When we decide to change the display name of a chain, we add the mapping for the new name here
+const newChainLabelMap = {
+  "zklink": "zkLink Nova",
+  "bitkub": "Bitkub Chain",
+  "lyra": "Derive Chain",
+  "fuel": "Fuel Ignition",
+  "op_bnb": "opBNB",
+  "hydradx": "Hydration",
+  "orai": "Oraichain",
+  "map": "MAP Protocol",
+  "umee": "UX",
+  "wemix": "WEMIX3.0",
+  "persistence": "Persistence One",
+  "ripple": "XRPL",
+  "milkomeda": "Milkomeda C1",
+  "zksync": "ZKsync Lite",
+  "zksync era": "ZKsync Era",
+  "tomochain": "Viction",
+  "elrond": "MultiversX",
+  "rsk": "Rootstock",
+  "pulse": "PulseChain",
+  "sapphire": "Oasis Sapphire",
+  "oasis": "Oasis Emerald",
+  "klaytn": "Kaia",
+  "kcc": "KCC",
+  "okexchain": "OKTChain",
+  "xdai": "Gnosis",
+  "cosmos": "CosmosHub",
+  "terra": "Terra Classic",
+  "bsc": "BSC",
+} as { [key: string]: string }
+
+const allChainLabelMap = {
+  ...chainLabelMap,
+  ...newChainLabelMap,
+} as { [key: string]: string }
+
+// mapping from old chain name to new chain name
+const chainOldLabelToNewLabelMap = {} as { [key: string]: string }
+
+Object.entries(newChainLabelMap)
+  .filter(([key,]) => chainLabelMap[key])
```

Comment:
everything looks good but i dont fully understand the filtering here

Reply:
bit of a precaution, ideally, there should never be a case where a key is present in newChainLabel object but not in old, but this is to handle that case, ignore those keys.